azure/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "sshPublicKey": {
+            "value": "your public key goes here"
+        },
+        "numberOfInstances": {
+            "value": 4
```

Comment:
I guess this includes the coordinator. Any chance we can have a "NumberOfWorkers"?

Reply:
Yeah, that would be more clear.